doc/devguide.md

```diff
@@ -119,7 +122,7 @@ Contains common code shared between multiple components. This doesn't generate a
 `/test` contains end-to-end tests run before each PR integration by our internal CI system.
 `/stress` contains SDK stress-test applications.
 
-### /jenkins
+### /jenkins and /vsts
 
 Contains scripts used by our internal Continuous Integration system (Jenkins).
```

Comment:
Do we still use Jenkins for CI?

Reply:
No: good point - will remove the Jenkins ref.